_posts/2021-01-01-Fortran-Newsletter-January-2021.md

```diff
@@ -0,0 +1,228 @@
+* [#315](https://github.com/fortran-lang/fpm/pull/315):
+  Remove: -ffast-math in gfortran default release flags
+* [#322](https://github.com/fortran-lang/fpm/pull/322):
+  Group sources by package in the model
+
+Work in progress:
+
+* [First beta release](https://github.com/fortran-lang/fpm/milestone/1) (WIP):
+  First feature-complete release of the Fortran implementation.
+* [#230](https://github.com/fortran-lang/fpm/pull/230),
+  [#261](https://github.com/fortran-lang/fpm/pull/261) (WIP):
+  Specification of the fpm CLI.
+* [#316](https://github.com/fortran-lang/fpm/pull/316) (WIP):
+  Update subcommand "new" to reflect the addition of support for examples
+
+`fpm` is still in early development and we need as much help as we can get.
+Here's how you can help today:
+
+* Use it and let us know what you think! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm, and the [manifest reference](https://github.com/fortran-lang/fpm/blob/master/manifest-reference.md) to learn what are all the things that you can specify in the fpm.toml file.
+* Browse existing *fpm* packages on the [fortran-lang website](https://fortran-lang.org/packages/fpm)
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm and submit it to the [Registry](https://github.com/fortran-lang/fpm-registry).
+* Improve the documentation.
+
+The short-term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Compilers
+
+### Classic Flang
+
+TO DO
+
+The Classic Flang biweekly call has been set up to discuss issues and plans
+for the next pull requests to be validated and merged. Our next calls are Wednesday, December 16 and 30, 8:00 AM Pacific time. The notes from previous calls, upcoming agenda and a link to join the call can be found
+[here](https://docs.google.com/document/d/1-OuiKx4d7O6eLEJDBDKSRnSiUO2rgRR-c2Ga4AkrzOI).
+
+### LLVM Flang
+
+TO DO
+
+
+## Events
+
+* We had our 7th Fortran Monthly call on December 15.
+You can watch the recording below:
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/S_xQCSRlefE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+As usual, subscribe to the [mailing list](https://groups.io/g/fortran-lang) and/or
+join the [Discourse](https://fortran-lang.discourse.group) to stay tuned with the future meetings.
+
+* [Episode 6](https://adspthepodcast.com/2021/01/01/Episode-6.html) of the
+Algorithms+Data Structures=Programs (ADSP) Podcast discussed Fortran and
+the recent fortran-lang developments.
```

Comment:
Nice! I was not aware of this podcast! The community 's efforts seem to pay off.